### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by chenghung**

### CODE REVIEW

**Comment by chenghung** [comment]
@koshuang r?, thanks.

**Comment by koshuang** [comment]
@chenghung r+, thanks!

File: README.md [hunk 1]
```diff
@@ -27,6 +27,35 @@ Node.js web developer is now easily to start developing web application in minut
 * Debugger (node-inspector)
 * Support Background Jobs (Kue.js)
 
+# Install
+
+
+```shell
+> npm install -g yarn
```
Comment (koshuang) [inline README.md:34]: missing `bower`?

Comment (chenghung) [reply README.md:34]: bower installation is added.

**Comment by koshuang** [comment]
@chenghung r+, thanks!

**Pull request merged**